libs/application/templates/estate/src/fields/EstateMembersRepeater/index.tsx

```
@@ -204,29 +203,78 @@
+              ) : (
+                <>
+                  <GridColumn span={['1/1', '1/2']} paddingBottom={2}>
+                    <InputController
+                      id={`${id}[${index}].nationalId`}
+                      name={`${id}[${index}].nationalId`}
+                      label={formatMessage(m.inheritanceKtLabel)}
+                      defaultValue={formatNationalId(member.nationalId || '')}
+                      backgroundColor="white"
+                      disabled={!member.enabled}
+                      format={'######-####'}
+                      error={error && error[index] && error[index].nationalId}
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)